ansible/roles/vmware-vm/tasks/post_deploy.yml

```diff
@@ -11,7 +11,7 @@
     folder: "{{ datacenter_name }}/vm/{{ testenv_folder }}"
     name: "{{ inventory_hostname_short }}"
   register: newvm
-  retries: 15
+  retries: 600
```

Comment:
Let's do one hour for now (perhaps 1.5h) - we think that if after one hour it doesn't have an IP, the problem is somewhere else.

(Also, some jobs as `deploy-demo-env` use this ansible role without any uppper timeout)